colcon_core/task/python/test/pytest.py

```diff
@@ -39,7 +39,8 @@ def add_arguments(self, *, parser):  # noqa: D102
             nargs='*', metavar='*', type=str.lstrip,
             help='Pass arguments to pytests. '
             'Arguments matching other options must be prefixed by a space,\n'
-            'e.g. --pytest-args " --help"')
+            'e.g. --pytest-args " --help" (stdout output might not be shown '
```

Comment:
Nit: I'd rather see either "standard output" or just "stdout".

Reply:
Good point: cc2934c364dd5a63dfc6941f8fceb26722edfdf2.

Reply:
Same in 991747f05a20c675ec492983e234d5a6519e2139.